packages/datagateway-common/src/card/entityCard.component.tsx

```diff
@@ -76,9 +79,11 @@ const useCardStyles = makeStyles((theme: Theme) => {
     shadowVisible: {
       position: 'absolute',
       height: 30,
-      minWidth: '30vw',
+      minWidth: '100%',
       top: 130,
-      background: 'linear-gradient(rgba(255, 255, 255, 0), #fff)',
+
+      //RGBA values seem to work best with linear gradients being dynamically changed by dark mode preference
```

Comment:
Is there a reason we need to use RGBA values rather than HEX codes? Would seem simpler if we could but this comment implies that RGBA is better

Reply:
I tried the HEX equivalent by adding alpha codes on the end of the values but it just didn't seem to work. I'll try one more time when I address your other comment with another commit

Reply:
I think we should be able to use alpha values with hex codes directly: see https://caniuse.com/css-rrggbbaa

Whether we don't for browser incompatibility is another issue (looking at you IE 11...)

Reply:
Yes I just got it working (was another issue causing the alpha values to fail). Fix inbound!